[2022-06-27 13:30:50,918] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dag1.model_A scheduled__2022-01-31T00:00:00+00:00 [queued]>
[2022-06-27 13:30:50,931] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dag1.model_A scheduled__2022-01-31T00:00:00+00:00 [queued]>
[2022-06-27 13:30:50,932] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-06-27 13:30:50,933] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-06-27 13:30:50,934] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-06-27 13:30:51,016] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): model_A> on 2022-01-31 00:00:00+00:00
[2022-06-27 13:30:51,036] {standard_task_runner.py:52} INFO - Started process 56901 to run task
[2022-06-27 13:30:51,062] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'dag1', 'model_A', 'scheduled__2022-01-31T00:00:00+00:00', '--job-id', '139', '--raw', '--subdir', 'DAGS_FOLDER/dag1.py', '--cfg-path', '/tmp/tmp8r54uj_j', '--error-file', '/tmp/tmpqt855ygw']
[2022-06-27 13:30:51,075] {standard_task_runner.py:80} INFO - Job 139: Subtask model_A
[2022-06-27 13:30:51,258] {task_command.py:370} INFO - Running <TaskInstance: dag1.model_A scheduled__2022-01-31T00:00:00+00:00 [running]> on host 43f8d36cfb0f
[2022-06-27 13:30:51,476] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=dag1
AIRFLOW_CTX_TASK_ID=model_A
AIRFLOW_CTX_EXECUTION_DATE=2022-01-31T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-01-31T00:00:00+00:00
[2022-06-27 13:30:51,486] {python.py:173} INFO - Done. Returned value was: 2
[2022-06-27 13:30:51,605] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=dag1, task_id=model_A, execution_date=20220131T000000, start_date=20220627T133050, end_date=20220627T133051
[2022-06-27 13:30:51,669] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-06-27 13:30:51,844] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
